README.md

```diff
@@ -48,59 +49,168 @@ For .NET Core CLI, use the following command:
+A Deferred Session in Alfred is a mechanism designed for asynchronous file uploads, integral to the document processing workflow. It serves as a container or grouping for files that are uploaded at different times or from various sources, but are all part of a single processing task or Job.
 
-#### Authentication Methods
+### Create a new Deferred Session
 
-To obtain the necessary credentials for the following authentication methods, please refer to the [Alfred API documentation](https://docs.tagshelf.dev/authentication) or contact the Alfred support team.
 
-The following examples demonstrate how to initialize the Alfred client with different authentication methods:
 
-   For OAuth authentication, specify the method and credentials explicitly and provide the environment type (Production or Staging):
 
-   ```csharp
-   var alfred = await AlfredFactory.CreateWithOAuthAsync("username", "password", EnvironmentType.Production);
-   ```
 
-   For HMAC authentication, provide the secret key, public key, and environment type (Production or Staging):
+### Retrieve Details of a Specific Deferred Session
 
-   ```csharp
-   var alfred = await AlfredFactory.CreateWithHmacAsync("secret_key", "public_key", EnvironmentType.Staging);
-   ```
 
-   For API key authentication, use the following method with the API key and environment type (Production or Staging):
 
-   ```csharp
-   var alfred = await AlfredFactory.CreateWithApiKeyAsync("api_key", EnvironmentType.Staging);
-   ```
 
-### File
 
-With the client initialized, you're ready to perform API operations.
 
-These steps demonstrate how to upload a file to Alfred using the `File` domain:
+## Files {#files-domain}
 
-1. Initialize the Alfred client.
-2. Choose the method to upload the file (from URL or stream).
-3. Create a request object with the necessary parameters.
-4. Call the appropriate method to upload the file.
-5. Handle the response accordingly.(The response will contain the file ID or the job ID, depending on the method used.)
+In Alfred, a file is an individual document or data unit undergoing specialized operations tailored for document analysis and management. It is processed by the platform for tasks such as classification, optical character recognition (OCR), and data extraction. Each file, as a distinct entity, transitions through various statuses, which mark its progress in the automated workflow. 
 
-#### Upload File (From URL or URLs)
 
-This method enables you to upload a file from one or multiple URLs to Alfred. During the upload process, you can also specify metadata for the file. Please note that using this method will automatically trigger job processing.
 
-   | Parameter | Type | Description |
+
+
+
+
+
+
+
+## Jobs {#jobs-domain}
+
+## Data Points {#data-points-domain}
+
+## Account {#account-domain}
```

Comment:
This title is empty. If we are not going to include information we should remove it entirely,